addons/mail/static/src/views/web/activity/activity_controller.js

```diff
@@ -57,6 +55,12 @@ export class ActivityController extends Component {
         };
     }
 
+    async _onPagerUpdate(params) {
```

Comment:
What about moving that method in the model (ActivityModel) ? Maybe as a method name "onUpdateParams" or something like that ? As this weird added domain would be fully handled by the model and not split across the controller and the model. What do you think ?

Reply:
Hello @abz-odoo ,

No that's also good. The advantage to move that method in the model would be to have no duplicate comment and all that weird logic at the same place.

But actually, I think you've forgot to remove the method _onPagerUpdate(params) which is no longer needed then if I'm not mistaken.